bin/ut-jenkins.js

```diff
@@ -13,9 +13,12 @@ var command;
 var jobname = process.env.JOB_NAME || '';
 
 if (
-    /((ut)|(impl))-.+_cr$/.test(jobname) &&
+    /^(ut|impl)-.+?(_cr|_post-commit)$/.test(jobname) &&
     process.env.BUILD_CAUSE === 'SCMTRIGGER' &&
-    (/(^master$)|(^ci_.*)/.test(process.env.gitlabSourceBranch) || /(^origin\/master$)|(^origin\/ci_.*)/.test(process.env.GIT_BRANCH))
+    (
+        /^(master|(fix|feat)\/[^/]+)$/.test(process.env.gitlabSourceBranch) ||
+        /^origin\/(master|(fix|feat)\/[^/]+)$/.test(process.env.GIT_BRANCH)
```

Comment:
Actually not because we should allow branches with the following names only:
origin/master
origin/fix/abc
origin/feat/abc

we don't want 'origin/master/abc' to be valid